docs/validator-guide/attestation-service.md

```diff
@@ -36,7 +37,10 @@ This guide steps you through setting up an Attestation Service:
 
 ## Recent releases
 
-- [Attestation Service v1.2.0](https://github.com/celo-org/celo-monorepo/releases/tag/attestation-service-v1.2.0) (latest production release)
+- [Attestation Service v1.5.0](https://github.com/celo-org/celo-monorepo/releases/tag/attestation-service-v1.5.0) (latest production release)
```

Comment:
@eelanagaraj Thanks for including this! Just came across this while browsing the docs and remembered Cody's Slack message. Appreciate you updating this :)